src/components/landing/Header.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import H2 from '@material-tailwind/react/Heading2';
+import LeadText from '@material-tailwind/react/LeadText';
+
+export default function Header() {
+    const backgroundImage =
+        'https://images.unsplash.com/photo-1557804506-669a67965ba0?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1267&q=80';
+    return (
+        <div className="relative pt-16 pb-32 flex content-center items-center justify-center h-screen">
+            <div
+                className="absolute top-0 w-full h-full"
+                style={{
```

Comment:
The background position and background sizes can be changed from inline style to tailwind css classes:
- https://tailwindcss.com/docs/background-position
- https://tailwindcss.com/docs/background-size
If they do not exist there, add them inside the tailwind config - i.e. extend the classes.